docs/source/integrated_network.rst

```diff
@@ -0,0 +1,61 @@
+.. _network-section:
+
+Create an Integrated Transit and Street Network
+=================================================
+
+After loading the transit and street network data, UrbanAccess has tools to compute a travel time weighted graph network of nodes and edges for use with Pandana network accessibility queries.
+
+.. _network-data-object:
+
+The Network Data Object
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The network data object stores the individual transit and street network node and edge components as well as the final integrated network in Pandas dataframes.
+
+.. autodata:: urbanaccess.network.ua_network
```

Comment:
Here and two lines below, also good candidates to use `:annotation:` to only show the name.